src/main/java/org/phyloref/jphyloref/helpers/ReasonerHelper.java

```diff
@@ -63,10 +64,12 @@ public static String getReasonerNameAndVersion(OWLReasonerFactory factory) {
 			Version version = reasoner.getReasonerVersion();
 			versionString = version.getMajor() + "." + version.getMinor() + "." + version.getBuild() + "." + version.getPatch();
 		} catch (OWLOntologyCreationException e) {
-			versionString = "(undefined)"; 
-		}		
-		
-		return factory.getReasonerName() + "/" + versionString; 
+			versionString = "(undefined)";
+		} catch (UnsatisfiedLinkError err) {
+			versionString = "(JNI library not linked)";
+		}
```

Comment:
So this in essence swallows the exception. I assume that's fine but whenever someone does that I always appreciate a comment indicating why that's OK, which prevent future guessing as to whether this was accidental or intentional.